code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1513,17 +1513,27 @@ func (srh *StorageRestHandler) getWriteMarker(w http.ResponseWriter, r *http.Req
 //      description: restrict to transactions in indicated block
 //      in: query
 //      type: string
+//    + name: start_block_number
+//      description: restrict to transactions in specified start block and endblock
+//      in: query
+//      type: string
+//    + name: end_block_number
+//      description: restrict to transactions in specified start block and endblock
+//      in: query
+//      type: string
 //
 // responses:
 //  200: []Transaction
 //  400:
 //  500:
 func (srh *StorageRestHandler) getTransactionByFilter(w http.ResponseWriter, r *http.Request) {
 	var (
-		clientID     = r.URL.Query().Get("client_id")
-		offsetString = r.URL.Query().Get("offset")
-		limitString  = r.URL.Query().Get("limit")
-		blockHash    = r.URL.Query().Get("block_hash")
+		clientID      = r.URL.Query().Get("client_id")
+		offsetString  = r.URL.Query().Get("offset")
```

Comment:
use common api for pagination, to be merged

Reply:
Done, let's merge these new api updates after `fix/benchmarks_loaddata`